distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -789,15 +789,15 @@ type DB_Table
        > Example
          Drop first 10 rows of the table.
 
-             table.drop (First 10)
+             table.drop (..First 10)
```

Comment:
Given the _autoscoped constructors_ are now used in production code, we should find some time to invest into measuring and improving their peak performance.